server/chat-commands/moderation.ts

```diff
@@ -305,9 +311,10 @@ exports.commands = {
 		if (target.startsWith('https://')) target = target.slice(8);
 		if (target.startsWith(`${Config.routes.client}/`)) target = target.slice(Config.routes.client.length + 1);
 		if (target.startsWith('psim.us/')) target = target.slice(8);
-		user.tryJoinRoom(target, connection).then(ret => {
+		void user.tryJoinRoom(target as RoomID, connection).then((ret: boolean | void | null) => {
```

Comment:
```suggestion
		void user.tryJoinRoom(target as RoomID, connection).then(ret => {
```

There's no reason why this should need to be manually type-specified, unless something's wrong with `user.tryJoinRoom`. If it is, fix it there (or leave it errored if you can't figure out how).